x/ecocredit/client/query.go

```diff
@@ -415,17 +391,15 @@ regen q ecocredit batch C01-001-20200101-20210101-001 --limit 10
 	})
 }
 
-// QueryBalanceCmd returns a query command that retrieves the tradable and
+// QueryBatchBalanceCmd returns a query command that retrieves the tradable and
 // retired balances for a given credit batch and account address.
-func QueryBalanceCmd() *cobra.Command {
+func QueryBatchBalanceCmd() *cobra.Command {
 	return qflags(&cobra.Command{
-		Use:   "balance [batch_denom] [account]",
-		Short: "Retrieve the tradable and retired balances of the credit batch",
-		Long:  "Retrieve the tradable and retired balances of the credit batch for a given account address",
-		Example: `
-regen q ecocredit balance C01-001-20200101-20210101-001 regen1r9pl9gvr56kmclgkpjg3ynh4rm5am66f2a6y38
-		`,
-		Args: cobra.ExactArgs(2),
+		Use:     "batch-balance [batch-denom] [account]",
```

Comment:
Although `balance` corresponds with the ecocredit module, we have basket and marketplace submodule commands that exist alongside the ecocredit commands so `batch-balance` made more sense in this context.

I think in general we should try to match the name of the command with the message or query name but we might need to rethink client architecture with submodules and probably best to do so alongside sdk v1 updates.